docs/docs-content/vm-management/architecture.md

```diff
@@ -0,0 +1,313 @@
+---
+sidebar_label: "Architecture"
+title: "Architecture"
+description: "Learn about Palette VMO pack and the architecture behind it."
+hide_table_of_contents: false
+sidebar_position: 0
+tags: ["VMO", "architecture"]
+---
+
+Palette Virtual Machine Orchestrator (VMO) is a composite solution that includes the Palette VMO pack and a variety of
+tools and technologies for deploying and managing Virtual Machines (VMs) alongside containerized applications.
+
+This page offers a reference architecture you can implement to create a standard, repeatable, and highly scalable VMO
+solution that can be further adapted and customized to fit specific environments.
+
+![Diagram that explains the architecture behind Palette VMO.](/vm-mangement_architecture_arch-diagram.webp)
```

Comment:
I like the diagram but I worry that the take away is that VMO only works with MAAS. You can use VMO with Edge and that's another important use case we support today.